packages/runtime/src/components/builtin/Image/Image.tsx

```diff
@@ -47,19 +48,24 @@ function loadImage(src: string): Promise<HTMLImageElement> {
   })
 }
 
-function imageSizes(width?: Props['width']): string {
-  const baseDevice = DEVICES.find(device => device.maxWidth == null)
-  const baseWidth = baseDevice && width && findDeviceOverride(width, baseDevice.id)
+function useImageSizes(width?: Props['width']): string {
```

Comment:
Note, there's no need to make _everything_ a hook. We can keep `imageSizes` and add a `breakpoints` arguments that replaces the `DEVICES` variable and then implement `useImageSizes` this way:

```ts
function useImageSizes(width) {
  const breakpoints = useBreakpoints()

  return imageSizes(width, breakpoints)
}
```

This is a general principle when writing React code. It's easy to shove everything into a hook, but then we lose the ability to reuse the `imageSizes` logic in other places. In this case it might not matter, but I've found myself repeating _a lot_ of code that was "embedded" in a hook this way when it didn't need to be